### PR TITLE
fix(controller): solve multi unit issues with continuos sync

### PR DESCRIPTION
**Pull request by Tbaile**
Ref:
- https://github.com/NethServer/nethsecurity/issues/638

### CODE REVIEW

**Review by gsanchietti — APPROVED** [review]
It looks good to me, but better wait Andrea approval

**Comment by Tbaile** [comment]
He's :100:% going to change the labels, which I agree they need change :smile:

File: src/components/controller/units/UnitsTable.vue [hunk 1]
```diff
@@ -77,61 +77,70 @@ async function openUnit(unitId: string) {
...
+    })
+  }
+  // apply common menu items
+  menuItems.push(
     {
-      id: 'openMetrics',
-      label: t('controller.units.open_metrics'),
-      icon: 'arrow-up-right-from-square',
+      id: 'refreshUnitInfo',
+      label: t('controller.units.refresh_unit_info'),
```
Comment (andre8244) [inline src/components/controller/units/UnitsTable.vue:120]:
```suggestion
      label: t('controller.units.sync_unit_info'),
```

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -1913,7 +1913,10 @@
       "confirm_remove_unit": "You are about to remove unit '{name}'. The unit will continue to operate normally, but it will no longer be managed by this Controller.",
       "unit_name_removed": "Unit '{name}' removed",
       "num_units_left": "No units left | {num} unit left | {num} units left",
-      "num_units_left_tooltip": "You have configured {num} of {max} units. Upgrade to a subscription plan to manage more units."
+      "num_units_left_tooltip": "You have configured {num} of {max} units. Upgrade to a subscription plan to manage more units.",
+      "sync_unit_info": "Sync unit info",
+      "syncing_data": "Syncing data...",
+      "syncing_data_description": "The data hasn't been updated yet. You can force the reload on the three dots menu on the right."
```
Comment (andre8244) [inline public/i18n/en/translation.json:1919]:
```suggestion
      "syncing_data_description": "Unit data hasn't been synced yet. You can do it now by clicking the three dots menu and then '@:controller.units.sync_unit_info'."
```

**Review by andre8244 — CHANGES_REQUESTED** [review]
Just a refererce to a wrong i18n string and some relabeling

**Review by andre8244 — APPROVED** [review]

**Pull request merged**